src/app/api/routes/alerts.py

```diff
@@ -81,15 +81,18 @@ async def create_alert_from_device(
     Below, click on "Schema" for more detailed information about arguments
     or "Example Value" to get a concrete idea of arguments
     """
-
     return await create_alert(AlertIn(**payload.dict(), device_id=device.id), background_tasks)
 
 
 @router.get("/{alert_id}/", response_model=AlertOut, summary="Get information about a specific alert")
-async def get_alert(alert_id: int = Path(..., gt=0), _=Security(get_current_access, scopes=[AccessType.admin])):
+async def get_alert(alert_id: int = Path(..., gt=0),
+                    requester=Security(get_current_access,
+                    scopes=[AccessType.admin, AccessType.user])):
     """
     Based on a alert_id, retrieves information about the specified alert
     """
+    requested_group_id = await get_entity_group_id(alerts, alert_id)
+    await check_group_read(requester.id, requested_group_id)
     return await crud.get_entry(alerts, alert_id)
```

Comment:
over these 3 lines, we are querying the alerts table twice, we should try to improve this I think

Reply:
Only way I see to slightly speed up the process would be to do 
```python

    retrieved_alert = await crud.get_entry(alerts, alert_id)
    requested_group_id = await get_entity_group_id(devices, retrieved_alert["device_id"])
    await check_group_read(requester.id, requested_group_id)
    return retrieved_alert
```